src/main/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtils.java

```diff
@@ -79,19 +83,97 @@
      */
     @Nonnull
     public static <T extends Custom & Resource<T>, S extends CustomDraft> List<UpdateAction<T>>
+        buildResourceCustomUpdateActions(
```

Comment:
Pls, "shortify" the comment for this method referring to the main implementation (`CustomUpdateActionUtils#buildCustomUpdateActions()), just specify this implementation is a sugar for `Resource` inherited classes (Products, Categories, Inventories)

Reply:
9c24f2a